src/components/Payment/BraintreeDropInInterface.vue

```diff
@@ -202,25 +202,29 @@ export default {
 <style lang="scss" scoped>
 @import "settings";
 
-$form-border-radius: rem-calc(2);
+$form-border-radius: rem-calc(4);
 $active-border-color: $input-checked-color;
 $icon-background-color: $input-checked-color;
 $border-width: 1px;
 
-#dropin-container {
-	::v-deep [data-braintree-id="choose-a-way-to-pay"] {
-		font-size: $small-text-font-size;
-	}
+// !TODO Change uses of $umbreon here for the functional name.
```

Comment:
Not sure what the functional name for $umbreon will be

Reply:
I believe it's meant to be $kiva-glyph-primary-black, which is replacing $kiva-text-dark